.gitignore

```diff
@@ -0,0 +1,4 @@
+#wsmux
+wsmux/log_session_*
```

Comment:
is this smart... if tests produce temporary files just create a randomly named folder in `os.TempDir()`... and do `defer os.RemoveAll(...)`..

There is even:
https://golang.org/pkg/io/ioutil/#TempDir
To create temporary folders...